src/context/auth-context.jsx

```diff
@@ -18,23 +18,14 @@ const AuthProvider = ({ children }) => {
   const [loading, setLoading] = useState(false);
   const [isNewUser, setIsNewUser] = useState(true);
 
-  useEffect(() => {
-    const authStateChange = onAuthStateChanged(auth, (res) => {
-      res ? setUser(res) : setUser(null);
-      setError("");
-    });
-
-    return authStateChange;
-  }, []);
-
   const signupUser = async (name, email, password) => {
     try {
       setLoading(true);
       const res = await createUserWithEmailAndPassword(auth, email, password);
       const updateUserProfile = await updateProfile(auth.currentUser, {
         displayName: name,
       });
-      localStorage.setItem("userID", JSON.stringify(res.user.uid) )
+      localStorage.setItem("userID", JSON.stringify(res.user.uid));
```

Comment:
Since `uid` is itself is a string, you need not `stringify` it

Reply:
Noted, have made the changes in the next PR